qcodes/instrument_drivers/oxford/IPS120.py

```diff
@@ -292,11 +310,8 @@ def _set_remote_status(self, mode):
             2: "Local and unlocked",
             3: "Remote and unlocked",
         }
-        if status.__contains__(mode):
-            log.info('Setting remote control status to %s' %
-                status.get(
-                    mode,
-                    "Unknown"))
+        if mode in status:
+            log.info('Setting remote control status to %s' % status[mode])
```

Comment:
I think this should be `self.log` See https://qcodes.github.io/Qcodes/examples/Creating%20Instrument%20Drivers.html#Logging

Reply:
Done.